.github/workflows/test-and-release.yml

```
@@ -318,28 +318,29 @@ jobs:
 
     - name: Get Yarn cache directory
       id: yarn-cache-dir-path
-      run: echo "::set-output name=dir::$(yarn cache dir)"
+      run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
 
     - name: Use Yarn cache
       uses: actions/cache@v2
       id: yarn-cache
       with:
         path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-        key: ${{ runner.os }}-yarn-${{ matrix.node-version }}-${{ hashFiles('**/yarn.lock') }}
+        key: yarn-${{ hashFiles('**/yarn.lock') }}
         restore-keys: |
-          ${{ runner.os }}-yarn-
+          yarn-
 
     - name: Install dependencies
-      run: yarn install --prefer-offline --frozen-lockfile
+      run: yarn install --immutable
 
     - name: Create a clean build
       run: yarn run build:full
 
-    - name: Publish package to npm
+    - name: Publish packages to npm
       run: |
-        npm config set //registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}
-        npm whoami
-        npx lerna publish from-package --yes
+        yarn config set npmPublishRegistry 'https://registry.npmjs.org'
+        yarn config set 'npmRegistries["//registry.npmjs.org"].npmAuthToken' "${{ secrets.NPM_TOKEN }}"
+        yarn npm whoami
+        yarn workspaces foreach exec npm publish --tolerate-republish
```

```suggestion
        yarn workspaces foreach npm publish --tolerate-republish
```

Otherwise it will use `npm` to do the publish

---

Yeah thanks. Was late last night, I forgot to remove that.